library/alignment.h

```diff
@@ -223,10 +240,10 @@ static const uint16_t mbedtls_byte_order_detector = { 0x100 };
  *                  integer from.
  */
 #define MBEDTLS_GET_UINT32_BE(data, offset)                              \
-    ((MBEDTLS_IS_BIG_ENDIAN)                                            \
+        ((MBEDTLS_IS_BIG_ENDIAN)                                            \
         ? mbedtls_get_unaligned_uint32((data) + (offset))                  \
         : MBEDTLS_BSWAP32(mbedtls_get_unaligned_uint32((data) + (offset))) \
-    )
+        )
```

Comment:
I don't remember all this whitespace change in `alignment.h` - what's happened?

Reply:
The coding style change happened. Looks like it didn't go right? @daverodgman can you please share exactly what command you ran and the `--verbose` output?

Reply:
Wrong uncrustify version I think